src/qt/dashstrings.cpp

```diff
@@ -367,7 +367,6 @@ QT_TRANSLATE_NOOP("dash-core", "Connect to a node to retrieve peer addresses, an
 QT_TRANSLATE_NOOP("dash-core", "Connection options:"),
 QT_TRANSLATE_NOOP("dash-core", "Copyright (C)"),
 QT_TRANSLATE_NOOP("dash-core", "Corrupted block database detected"),
-QT_TRANSLATE_NOOP("dash-core", "Could not parse masternode.conf"),
```

Comment:
This file is autogenerated, should use `cd src/ && make translate` to update it. Or better keep it as is for now, it's usually a part of "update translations" PRs.

Reply:
Removed the changes in this file